src/slick.grid.ts

```diff
@@ -3785,7 +3785,7 @@ export class SlickGrid<TData = any, C extends Column<TData> = Column<TData>, O e
     return item[columnDef.field as keyof TData];
   }
 
-  protected appendRowHtml(stringArrayL: string[], stringArrayR: string[], row: number, range: CellViewportRange, dataLength: number) {
+  protected appendRowHtml(stringArrayL: HTMLElement[], stringArrayR: HTMLElement[], row: number, range: CellViewportRange, dataLength: number) {
```

Comment:
@JesperJakobsenCIM could you also rename all these `stringArray` to maybe `elementArray` or something else because it's no longer a string array so it's kinda confusing to leave the old name.